src/main/groovy/info/solidsoft/gradle/pitest/PitestPluginExtension.groovy

```diff
@@ -90,6 +90,7 @@ class PitestPluginExtension {
     final SetProperty<String> excludedTestClasses
     final SetProperty<String> avoidCallsTo
     final Property<Boolean> verbose
+    final Property<String> verbosity
```

Comment:
Please add Javadoc mentioning possible values, the value used by default, minimal spported PIT version and the `@since` tag (for GPP 1.9.1).